alephmarcreader/abstractalephmarcreader.py

```diff
@@ -7,39 +7,67 @@
 
 class AbstractAlephMarcReader(ABC):
 
-    class Person:
+    class Correspondent(ABC):
+        """
+        Represents an abstract correspondent.
+        Both Persons and Organisations can be Correspondents.
+        :param str|False name: the name of the correspondent, if any.
+        :param str|False gnd: the GND of the correspondent, otherwise 'no_GND', if any.
+        :param str|False roles: the role of the correspondent (author etc.), if any.
+
+        """
+
+        CORRESPONDENT_TYPE_PERSON = 'person'
```

Comment:
This should have a capital "p" like the class name.

Reply:
done